db-server/lib/error.js

```diff
@@ -83,17 +83,6 @@ AppError.invalidVerificationMethod = function () {
   )
 }
 
-AppError.unknownDeviceCapability = function () {
-  return new AppError(
-    {
-      code: 400,
-      error: 'Bad request',
-      errno: 139,
-      message: 'Unknown device capability'
-    }
-  )
-}
```

Comment:
@eoger points out that we don't have any code depending on this errno, so it's safe to just remove it rather than deprecating it.